cargo-dist/src/errors.rs

```diff
@@ -42,6 +42,10 @@ pub enum DistError {
     #[error(transparent)]
     FromUtf8Error(#[from] std::string::FromUtf8Error),
 
+    #[cfg(cli)]
+    #[error(transparent)]
+    AxopupdaterError(#[from] axoupdater::AxoupdateError),
```

Comment:
typo of the variant

Reply:
Whoops, the `cfg` weirdly hid it. Will fix.